<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtTestGUI</class>
 <widget class="QMainWindow" name="qtTestGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ezTestFramework</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Icons/Icons/pie.png</normaloff>:/Icons/Icons/pie.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTreeView" name="testTreeView">
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <attribute name="headerStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonQuit">
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAbort">
        <property name="text">
         <string>Abort</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRunTests">
        <property name="text">
         <string>Run Tests</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="testStatusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionRunTests"/>
    <addaction name="separator"/>
    <addaction name="actionAbort"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionAssertOnTestFail"/>
    <addaction name="actionOpenHTMLOutput"/>
    <addaction name="actionKeepConsoleOpen"/>
    <addaction name="actionShowMessageBox"/>
   </widget>
   <addaction name="menuRun"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionAssertOnTestFail">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Assert on Test Fail</string>
   </property>
  </action>
  <action name="actionOpenHTMLOutput">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open HTML Output</string>
   </property>
  </action>
  <action name="actionKeepConsoleOpen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Keep Console Open</string>
   </property>
  </action>
  <action name="actionRunTests">
   <property name="text">
    <string>Run Tests</string>
   </property>
   <property name="toolTip">
    <string>Run Tests</string>
   </property>
   <property name="shortcut">
    <string>Alt+R</string>
   </property>
  </action>
  <action name="actionAbort">
   <property name="text">
    <string>Abort</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Alt+Q</string>
   </property>
  </action>
  <action name="actionEnableOnlyThis">
   <property name="text">
    <string>Enable Only This</string>
   </property>
   <property name="toolTip">
    <string>Disables all tests and enables only this test.</string>
   </property>
  </action>
  <action name="actionEnableAll">
   <property name="text">
    <string>Enable All</string>
   </property>
   <property name="toolTip">
    <string>Enables all tests.</string>
   </property>
  </action>
  <action name="actionDisableAll">
   <property name="text">
    <string>Disable All</string>
   </property>
   <property name="toolTip">
    <string>Disables all tests.</string>
   </property>
  </action>
  <action name="actionExpandAll">
   <property name="text">
    <string>Expand All</string>
   </property>
   <property name="toolTip">
    <string>Expands all test nodes.</string>
   </property>
  </action>
  <action name="actionCollapseAll">
   <property name="text">
    <string>Collapse All</string>
   </property>
   <property name="toolTip">
    <string>Collapses all test nodes.</string>
   </property>
  </action>
  <action name="actionEnableAllChildren">
   <property name="text">
    <string>Enable All Children</string>
   </property>
   <property name="toolTip">
    <string>Enables all children of the selected test.</string>
   </property>
  </action>
  <action name="actionShowMessageBox">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show MessageBoxes</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonAbort</sender>
   <signal>clicked()</signal>
   <receiver>actionAbort</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>665</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonQuit</sender>
   <signal>clicked()</signal>
   <receiver>actionQuit</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>665</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRunTests</sender>
   <signal>clicked()</signal>
   <receiver>actionRunTests</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>665</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
